test/index.test.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+/* eslint-disable no-console*/
+const assert = require('chai').assert;
+const sinon = require('sinon');
+const mockery = require('mockery');
+
+sinon.assert.expose(assert, { prefix: '' });
+
+const TEST_XML =
```

Comment:
How is this different from text-job.xml? Can you just load from that file as a fixture instead?